examples/example-02-ruby-sample/tests/integration_test.rs

```diff
@@ -0,0 +1,49 @@
+use libcnb_test::IntegrationTest;
+use std::io;
+use std::io::{Read, Write};
+use std::net;
+use std::net::ToSocketAddrs;
+use std::time::Duration;
+
+#[test]
+fn basic() {
+    IntegrationTest::new("heroku/buildpacks:20", "test-fixtures/simple-ruby-app").run(|context| {
+        assert!(context.pack_stdout.contains("---> Ruby Buildpack"));
+        assert!(context.pack_stdout.contains("---> Installing bundler"));
+        assert!(context.pack_stdout.contains("---> Installing gems"));
+
+        context.start_container(&[12345], |container| {
+            std::thread::sleep(Duration::from_secs(1));
+
+            assert_eq!(
+                call_test_fixture_service(
+                    container.address_for_port(12345).unwrap(),
+                    "Hello World!"
+                )
+                .unwrap(),
+                "!dlroW olleH"
+            );
+
+            assert!(container
+                .shell_exec("ruby --version")
+                .stdout
+                .contains("ruby 2.7.0p0"));
+        });
+    });
+}
+
+fn call_test_fixture_service<A>(a: A, payload: impl AsRef<str>) -> io::Result<String>
+where
+    A: ToSocketAddrs,
+{
+    let mut stream = net::TcpStream::connect(a).unwrap();
+
+    stream
+        .write_all(format!("{}\n", payload.as_ref()).as_bytes())
+        .unwrap();
+
+    let mut buffer = vec![];
+    stream.read_to_end(&mut buffer).unwrap();
+
+    Ok(format!("{}", String::from_utf8_lossy(&buffer)))
```

Comment:
Good catch! Fixed in 519d2ae

Reply:
Clippy's pedantic lint `clippy::unnecessary_wraps` would have caught this. It seems that since integration tests are compiled separately, they don't inherit the clippy enable annotations from `main.rs`. In addition, we don't even have those annotations in the example `main.rs` files anyway.

I've filed #302 for this.